tools/generate-update-graph/main.go

```diff
@@ -41,6 +41,9 @@ func main() {
 
 func postgresChannel() updates.Channel {
 	releases := []updates.State{
+		{ID: "v1.20.0", Tag: "v1.20.0", Migration: "add-gc-covering-index"},
+		{ID: "v1.19.0", Tag: "v1.19.0", Migration: "add-gc-covering-index"},
+		{ID: "v1.19.0", Tag: "v1.19.0", Migration: "add-caveat-covering-index"},
```

Comment:
@ecordell v1.19.0 adds 2 postgres migrations. I added them in descending order, assuming that the operator respects the order of migrations within a version. Is that assumption correct?

Reply:
We'd typically just put the newest migration for the version (i.e. what you'd get to if you ran `migrate head` from that release). The only exception is phased migrations where we need to stop at a particular version for some period of time.